lib/build.gradle.kts

```diff
@@ -68,13 +71,72 @@ dependencies {
 
 afterEvaluate {
     publishing {
+        val version = "0.1"
```

Comment:
As we discussed, see if we can add a task which create a tag based on the version defined here